core/src/main/java/org/elasticsearch/search/SearchModule.java

```diff
@@ -245,14 +243,7 @@ protected void configureFetchSubPhase() {
     }
 
     protected void configureSuggesters() {
```

Comment:
This whole method is now only useful for backwards compatibility. I don't really think this is needed any more.